packages/venia-concept/src/components/ProductImageCarousel/carousel.js

```diff
@@ -1,6 +1,6 @@
-import React, { Component } from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
```

Comment:
Can you update this to import the specific `propTypes`:

```suggestion
import { arrayOf, bool, number, shape, string } from 'prop-types';
```